setup.py

```diff
@@ -30,6 +30,7 @@ def read_version(package):
     'inflection>=0.3.1',
     'pathlib>=1.0.1; python_version < "3.4"',
     'typing>=3.6.1; python_version < "3.6"',
+    'testfixtures>=5.3.0'
```

Comment:
this will pull in `testfixtures` everywhere, but we only need it for tests

Reply:
Yup, sorry about that. Added it in the `requirements-devel.txt`